### ISSUE TITLE
Integrate better NumPy matrix code

### ISSUE THREAD

**Issue by JelteF** [JelteF/PyLaTeX#1]
A simple way to convert NumPy matrices and arrays to LaTeX formatted matrices or vectors.

**Comment by Rsullivan00** [comment]
Is this already done? I see the Matrix and VectorName classes in numpy.py.

**Comment by JelteF** [comment]
Yes, the matrix and vector name classes work. I have kept the issue open because the code above hasn't been included in the project yet. 

You could check the example directory for the numpy example if you want to use it yourself.
